docs/src/examples/RunOnUI.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { View, StyleSheet, Text } from 'react-native';
-import {
+import Animated, {
   runOnJS,
   measure,
   useAnimatedRef,
   runOnUI,
 } from 'react-native-reanimated';
 
-function MeasurableText(props) {
+function MeasurableText(props: {children: any; onPress: (measurements: any) => void}) {
```

Comment:
We should avoid `any` in types declaration

Reply:
ok